### ISSUE TITLE
Accessibility: Panel should have aria-labelledby and aria-describedby like Dialog

### ISSUE THREAD

**Issue by ipip2005** [OfficeDev/office-ui-fabric-react#3634]
In order to let screen reader read something when Panel is opened, currently we can open leverage the "headerText" props. But it's only the title of the Panel.
Usually we have more text should be read like dialog description.

Panel should also provide such functionality for a11y like [Dialog does](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Dialog/Dialog.tsx#L56-L57)

**Comment by michaelangotti** [comment]
@betrue-final-final Can you comment on this one?
